### PR TITLE
Feature merchant/price routes

### PR DESCRIPTION
**Pull request by Kiwi-x-Kiwi**
- Added new Price and Merchant Models.
- Got back-end routes working for /api/prices

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ app.use('/api/reviews', require('./routes/reviews'))
 app.use('/api/posts', require('./routes/posts'))
 app.use('/api/comments', require('./routes/comments'))
 app.use('/api/games', require('./routes/games'))
+app.use('/api/prices', require('./routes/prices'))
+// app.use('/api/merchants', require('./routes/merchants'))
```
Comment (AndresMWeber) [inline server/app.js:52]: please remove or uncomment

File: server/models/Merchant.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const merchantSchema = new Schema({
+  url: String,
```
Comment (AndresMWeber) [inline server/models/Merchant.js]: please use validator.isURL

File: server/models/Merchant.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  name: String,
```
Comment (AndresMWeber) [inline server/models/Merchant.js]: Add required: true

File: server/models/Merchant.js (hunk 2 above)
Comment (AndresMWeber) [reply server/models/Merchant.js]: Add required: true

File: server/models/Merchant.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  country: String,
```
Comment (AndresMWeber) [inline server/models/Merchant.js]: Add required: true

File: server/models/Price.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const validator = require("validator")
+
+const priceSchema = new Schema({
+  game: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Game",
+    require: true,
```
Comment (AndresMWeber) [inline server/models/Price.js]: typo: should be required.

File: server/routes/prices.js [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+/**
+ * Get a specific price 
+ * @example 
+ * GET /api/prices/:id
+ */
+router.get('/:id', async (req, res, next) => {
+  try {
+    const price = await Price.findById(req.params.id)
+      .populate('game')
```
Comment (AndresMWeber) [inline server/routes/prices.js]: refactor to 'game merchant'

File: server/models/Price.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  merchant: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Merchant",
+    required: true,
+  },
+  price_text: {
```
Comment (AndresMWeber) [inline server/models/Price.js]: this should probably just be price, gets confusing...and redudant. or just text...but that seems to generic

**Review by AndresMWeber — CHANGES_REQUESTED** [review]

File: server/models/Price.js (hunk 7 above)
Comment (Kiwi-x-Kiwi) [reply server/models/Price.js]: Will change to price for now.

File: server/models/Merchant.js (hunk 3 above)
Comment (Kiwi-x-Kiwi) [reply server/models/Merchant.js]: Got it 👍

**Pull request merged**